hack/format-scripts.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+[[ -n "${DEBUG:-}" ]] && set -x
+
+gopath="$(go env GOPATH)"
+
+# install shfmt that ensures consistent formal in scripts
+if ! [[ -x "${gopath}/bin/shfmt" ]]; then
+  echo >&2 'Installing shfmt'
+  curl --silent --fail --location https://github.com/mvdan/sh/releases/download/v2.6.4/shfmt_v2.6.4_linux_amd64 >"${gopath}/bin/shfmt"
+  chmod u+x "${gopath}/bin/shfmt"
+fi
+
+"$gopath/bin/shfmt" -w -i=2 hack
```

Comment:
use SCRIPT_DIR and compute where exactly hack/ is (see other scripts).
if I call this after `cd hack` it won't work.

Reply:
done

but how about go part in lint, it would fail too right now.